src/constraints.jl

```diff
@@ -7,7 +7,7 @@ end
 
 function PopulationConstraint(graph::BaseGraph,
                               population_col::AbstractString,
-                              tolerance::Float64=0.01)
+                              tolerance::Float64)
```

Comment:
Thought it better to make the user consciously choose the tolerance than having a default.